assemblies/karaf-features/src/main/feature/feature.xml

```diff
@@ -658,7 +658,7 @@
   <feature name="opencast-security-cas" version="${project.version}">
     <bundle start-level="82">mvn:commons-collections/commons-collections/3.2.1</bundle>
     <bundle start-level="82">mvn:org.apache.velocity/velocity/1.7</bundle>
-    <bundle start-level="82">mvn:org.apache.santuario/xmlsec/2.1.2</bundle>
+    <bundle start-level="82">mvn:org.apache.santuario/xmlsec/2.2.6</bundle>
```

Comment:
Looks reasonable. Does it make sense to define this as `xmlsec.version` in the main `pom.xml` like we do it with a lot of other libraries?
https://github.com/opencast/opencast/blob/682fd48f857621d6a42442fb2e026a2e4962cbe3/pom.xml#L18-L20